src/index.js

```diff
@@ -49,29 +49,6 @@ import './images/water-icon.png'
 
 console.log('This is the JavaScript entry file - your code begins here.');
 
-
-const weeklyStepsChart = $('#weekly-steps-chart');
-const weeklyMinutesChart = $('#weekly-minutes-chart');
-const weeklyFlightsChart = $('#weekly-flights-chart');
-const weeklyOzGraph = $('#weekly-oz-graph');
-const name = $('#name');
-const address = $('#address');
-const email = $('#email');
-const strideLength = $('#strideLength');
-const dailyStepGoal = $('#stepGoal');
-const stepCompare = $('#step-compare');
-const dailyOz = $('#daily-oz');
-const date = $('#date');
-const yesterdaySleep = $('#yesterday-sleep');
-const weeklySleep = $('#weekly-sleep');
-const allTimeSleep = $('#all-time-sleep');
-const dailyActivity = $('#daily-activity');
-const compareActivity = $('#compare-activity');
-const friendSteps = $('#friend-weekly-steps');
-const stepTrends = $('#step-trends');
-const stepGoalChart = $('#step-goal-chart');
-const friendList = $('#friend-list');
-
```

Comment:
Really nice to see this cleaned up. I was never quite sure why this was necessary, as it seemed to add lines of code, rather than saving them.